studio/schemas/documents/compensations.ts

```diff
@@ -17,7 +18,13 @@ const compensations = defineType({
     defineField({
       name: "showSalaryCalculator",
       title: "Show Salary Calculator",
-      description: "Should the salary calculator be visible on the page?",
+      description: MultiLineDescription({
+        lines: [
+          "Should the salary calculator be visible on the page?",
+          "🇳🇴 Only for Norway",
```

Comment:
We don't need to add this. The creator knows whether or not they have the data needed for a calculator

```suggestion
```